src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,514 @@
+The included metadata are divided into sections described below.
+
+### PET sidecar JSON (`*_pet.json`)
+
+#### Information
+
+This section is mandatory and contains general information about the imaging experiment.
+Some of the fields are marked optional (MAY), for example anaesthesia; for those fields
+the BIDS validator will not throw an error even if they are not present.
+Note, although bodyweight is a recommended information in (Knudsen et al. 2020, JCBFM [1]),
+this consists of meta information at the participant level and should hence
+be part of the `participants.tsv` or `session.tsv` file in case of multiple measurements.
+
+| Field name                | Definition                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+|---------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Manufacturer              | REQUIRED. Scanner manufacturer (for example, "Siemens").                                                                                                                                                                                                                                                                                                                                                                                      |
+| ManufacturersModelName    | REQUIRED. PET scanner model name (for example, "mMR Biograph").                                                                                                                                                                                                                                                                                                                                                                               |
+| Unit                      | REQUIRED. Unit of the image file; please see BIDS main spec section 6. SI unit for radioactivity (Becquerel) should be used (for example, "Bq/mL"). Corresponds to DICOM Tag 0054, 1001 Units.                                                                                                                                                                                                                                                |
```

Comment:
```suggestion
| Units                      | REQUIRED. Unit of the image file; please see BIDS main spec section 6. SI unit for radioactivity (Becquerel) should be used (for example, "Bq/mL"). Corresponds to DICOM Tag 0054, 1001 Units.                                                                                                                                                                                                                                                |
```

I would say Unit**s** or PETUnits, usually it's allways plural

Reply:
Actually, I disagree. I think unit is used in signular and since we are talking about the SI unit Becquerel, it's also a single unit. Does that make sense?